pwa/package.json

```diff
@@ -10,9 +10,11 @@
     "format:check": "prettier --check .",
     "format:fix": "prettier --write .",
     "start": "remix-serve ./build/server/index.js",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "generate-api": "oazapfts ../src/backend/web/static/swagger/api_v3.json --optimistic --argumentStyle=object > app/api/v3.ts"
```

Comment:
Does this need to get regenerated after #6071 landed?

Reply:
yes

Reply:
done